.ci/test_r_package.sh

```diff
@@ -148,7 +148,6 @@ elif [[ $R_BUILD_TYPE == "cran" ]]; then
     # a change in a PR has changed configure.ac
     if [[ $OS_NAME == "linux" ]]; then
         ./R-package/recreate-configure.sh
-
```

Comment:
Please add this empty line back, unless it's something `shellcheck` was complaining about.